lib/util.py

```diff
@@ -327,6 +327,9 @@ def timeInDayLocalSeconds():
     return int(time.time() - sod)
 
 
+time_format_twentyfour = "M" not in xbmc.getInfoLabel('System.Time')
```

Comment:
This should probably be `xbmc.getInfoLabel('System.Time').upper()` for safety in case Kodi changes one day to lower case.

The one possible problem I see with this is that it only works for languages that have AM or PM. This probably isn't a huge issue since I think it's primarily English speaking countries that use a 12 hour time format anyway, and it's still an improvement, even if it could fail in some cases. Was `locale.use24hourclock` that you mentioned not usable?

Reply:
Hmm. You're right. Perhaps 2e72927 is the better solution together with `.upper()`.

Reply:
On the other hand - are there languages with 12h clocks without any indicator that it's AM or PM? Seem highly unlikely to me.

Reply:
`locale.use24hourclock` only returns `regional` if it's not specifically set to 12h or 24h by the user. I thought about using dateutil to get the correct time format for the current locale in that case, but that wouldn't have been better necessarily.